subworkflows/local/prepare_references.nf

```diff
@@ -101,6 +101,9 @@ workflow PREPARE_REFERENCES {
 
     ARRIBA_DOWNLOAD()
 
+    //cosmic_usr = params.cosmic_usr ?: ""
```

Comment:
i think these lines have been added back in by accident, please remove them. we are no longer using cosmic in the pipeline.